### PR TITLE
Merge boilerplate-app v2.1 changes

### PR DESCRIPTION
**Pull request by KingGorrin**
This PR merges the boilerplate-app changes up to v2.1.

- Fix fuzzing and complete implementation
- Update workflows from boilerplate-app v2.1
- Replace VSCode Tasks with ledger-dev-tools extension
- Fix possible overflow bug in `token_standard` and `address`
- Update version to 1.0.1
- Minor code clean-up

### CODE REVIEW

File: src/address.c [hunk 1]
```diff
@@ -40,27 +39,27 @@ bool address_from_pubkey(const uint8_t *public_key, char *out, size_t out_len) {
...
-    char bech32[73 + 1] = {0};
-
+    // Ensure the output buffer is large enough
     if (41 > out_len) {
         return false;
     }
 
+    char bech32[73];
+
     if (bech32_addr_encode(bech32, "z", prog, ADDRESS_LEN) != 1) {
```
Comment (georgezgeorgez) [inline src/address.c:57]: can you add a comment about what this is checking

Comment (KingGorrin) [reply src/address.c:63]: added comments to both functions and removed the explicit zero initialization of the hash variable.

Comment (georgezgeorgez) [reply src/address.c:63]: thanks looking again

**Comment by 0x3639** [comment]
> ## Welcome to [Codecov](https://codecov.io?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=hypercore-one) 🎉
> Once merged to your default branch, Codecov will compare your coverage reports and display the results in this comment.
> 
> Thanks for integrating Codecov - We've got you covered ☂️

Is this new?  I did not add this.

**Comment by KingGorrin** [comment]
> Is this new?

Not really, I registered a new account for this repo.

**Review by georgezgeorgez — APPROVED** [review]

**Pull request merged**